frontend/src/pages/StudentProfile.vue

```diff
@@ -77,23 +92,24 @@ export default {
     };
   },
   computed: {
-    ...mapGetters([ 'getStudentByStudentId' ])
+    ...mapGetters(["getStudentByStudentId"]),
   },
-  mounted: function() {
-    const id = this.$route.params.id
-    const data = this.getStudentByStudentId(id)
+  mounted: function () {
+    const id = this.$route.params.id;
+    const data = this.getStudentByStudentId(id);
 
     const studentObject = {
-          StudentID: data.StudentID,
-          PhoneNumber: data.PhoneNumber,
-          FullName: data.FullName,
-          Source: data.Source,
-          nativeLanguage: data.nativeLanguage,
-          status: data.status,
-          Notes: data.Notes,
-          dateJoined: new Date(data.created_at).toDateString(),
-          EnglishProficiency: data.EnglishProficiency,
-        };
+      StudentID: data.StudentID,
+      PhoneNumber: data.PhoneNumber,
+      FullName: data.FullName,
+      Source: data.Source,
+      nativeLanguage: data.nativeLanguage,
+      status: data.status,
+      Notes: data.Notes,
+      dateJoined: new Date(data.created_at).toDateString(),
+      EnglishProficiency: data.EnglishProficiency,
+    };
+
 
     const studentHistory = data.statusUpdates.map(update => {
       if (update.reason == null) {
```

Comment:
Is the Student Profile (e.g. `localhost:3000/students/1` working for you? I had to make the below change to get it working:

```suggestion
      update.reason = Object.assign([], update.reason)

      if (update.reason.Reason === undefined) {
```